renovate.json

```diff
@@ -35,6 +35,11 @@
       "matchPackagePatterns": ["eslint", "eslint-*", "@typescript-eslint/*"],
       "automerge": true
     },
+    {
+      "groupName": "React RC types - broken versions",
+      "matchPackageNames": ["@types/react-dom", "@types/react-is"],
```

Comment:
Disabled updating types for react-dom and react-is, as their versions on npm are messed up - 19.0.0 is older than 19.0.0-rc.1.